src/i18n/nb_NO.ts

```diff
@@ -0,0 +1,873 @@
+        },
+        11: {
+            caption: 'Politihelikopter',
+            color: '#ca1616',
+            coins: 30,
+            credits: 300_000,
+            minPersonnel: 1,
+            maxPersonnel: 2,
+            wtank: 0,
+            schooling: 'Police - Police Aviation',
+            shownSchooling: 'Police Aviation',
+        },
+        12: {
+            caption: 'Pansret kjøretøy',
+            color: '#a51212',
+            coins: 25,
+            credits: 17_300,
+            minPersonnel: 6,
+            maxPersonnel: 6,
+            wtank: 0,
+            schooling: 'Police - Våpentrening',
+            shownSchooling: 'SWAT',
+            special: 'Nødvendig etter at du har bygt 8 politistasjoner.',
+        },
+        13: {
+            caption: 'Hundepatrulje',
+            color: '#aa2222',
+            coins: 25,
+            credits: 7_000,
+            minPersonnel: 1,
+            maxPersonnel: 2,
+            wtank: 0,
+            schooling: 'Politi - Hundepatrulje',
+            shownSchooling: 'Hundepatrulje',
+            special: 'Nødvendig etter at du har bygt 6 politistasjoner.',
+        },
+        14: {
+            caption: 'Politimotorsykkel',
+            color: '#662222',
+            coins: 18,
+            credits: 2_500,
+            minPersonnel: 1,
+            maxPersonnel: 1,
+            wtank: 0,
+            schooling: 'Politi - Politimotorsykkel',
+            shownSchooling: 'Politimotorsykkel',
+        },
+        15: {
+            caption: 'Delta kjøretøy',
+            color: '#332222',
+            coins: 23,
+            credits: 7_000,
+            minPersonnel: 2,
+            maxPersonnel: 4,
+            wtank: 0,
+            special: 'Required once you have built 8 police stations',
+            schooling: 'Police - SWAT',
+            shownSchooling: 'SWAT',
+        16: {
+            caption: 'Røykdykkerbil',
+            color: '#aa0000',
+            coins: 25,
+            credits: 11_860,
+            minPersonnel: 1,
+            maxPersonnel: 3,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 5 brannstasjoner.',
+        },
+        17: {
+            caption: 'Innsatsstøttebil',
+            color: '#791515',
+            coins: 25,
+            credits: 25_500,
+            minPersonnel: 1,
+            maxPersonnel: 6,
+            wtank: 0,
+            schooling: 'Krever spesialistutdanning (Skadestedsledelse) ',
+            shownSchooling: 'Innsatsstøttebil',
+            special: 'Nødvendig etter at du har bygd 13 brannstasjoner.',
+        },
+        18: {
+            caption: 'Redningsbil',
+            color: '#880000',
+            coins: 25,
+            credits: 12_180,
+            minPersonnel: 1,
+            maxPersonnel: 4,
+            wtank: 0,
+            special: 'Nødvendig etter at du har bygd 4 brannstasjoner.',
+        },
+        19: {
+            caption: 'Lett redningsbåt',
```

Comment:
19 is Dykkerbil
20 is the boat

Reply:
Now corrected, after I discovered the list in https://github.com/LSS-Manager/lss-manager-v3/blob/master/lss-manager-v3/helperfunctions.js